cwltool/main.py

```diff
@@ -695,6 +697,18 @@ def main(argsl=None,  # type: List[str]
                 argsl = sys.argv[1:]
             args = arg_parser().parse_args(argsl)
 
+        # If On windows platform, A default Docker Container is Used if not explicitely provided by user
+        if onWindows() and not args.default_container:
+            args.default_container = "ubuntu"
+
+        # Making default Container as lowest priority, that should be used if nothing overrides it
+        if args.default_container:
```

Comment:
Yes, A little strange behaviour was shown. For a tool, docker was called, but our path changes that we do under a condition that docker is being called, didn't took place. My first guess is we extract info about docker requirement from `get_feature` function but since now we are not adding default container as hints this  condition return false. A new find_default_container function has been added in main file by some pr, but it wasn't called. I tried to trace it but couln't find a place from where it can be called(It is passed as arg to maketool). So for now I am sticking with this code.